runtime/queries/nginx/highlights.scm

```diff
@@ -0,0 +1,44 @@
+(comment) @comment
+
+(number) @number
+(metric) @number
+
+(regex) @regex
```

Comment:
Let's highlight this as a `string` and add an injection for it in injections.scm to use the regex language:

```scm
((regex) @injection.content
 (#set! injection.language "regex"))
```